src/cli.ts

```diff
@@ -200,6 +201,25 @@ async function main(): Promise<void> {
         await writeFile('package.json', JSON.stringify(packageJson, null, 2))
     }
 
+    console.log('📂 Creating dev directory')
+    await mkdir('dev')
+    await writeFile('dev/symlinkPackage.js', [
```

Comment:
I dislike that we are cluttering every single extension repo with this much script code. Is there really no other way to do this in an npm script? Is there no CLI for `create-symlink`? If not, can we just create one?

Reply:
https://www.npmjs.com/search?q=create%20symlink%20cli

Reply:
https://github.com/sourcegraph/create-extension/pull/43